picarto/channel_embed.py

```diff
@@ -124,13 +124,52 @@ def panel(self) -> tuple[str, str]:
         body = panel.body or "No information found"
 
         return title, body
+    
+    @property
+    def rules_panel(self) -> tuple[str, str]:
+        panels = self._channel_details.description_panels
+
+        panel: DescriptionPanel = sorted(panels, key=lambda p: p.position)[1]
+
+        title = panel.title or "Channel information"
+        body = panel.body or "No information found"
+
+        return title, body
+    
+    @property
+    def links_panel(self) -> tuple[str, str]:
+        panels = self._channel_details.description_panels
+
+        panel: DescriptionPanel = sorted(panels, key=lambda p: p.position)[2]
+
+        title = panel.title or "Channel information"
```

Comment:
Instead of "Channel information", maybe "Channel links" would be more clear? (That's what's shown in Discord if there's a problem)

Reply:
Got it! I shall change this string as specified